react-app/credentials.json

```diff
@@ -0,0 +1,17 @@
+{
+  "android": {
+    "keystore": {
+      "keystorePath": "/home/sai/Seated-Exercises-in-Vehicles/upload_keys/keystore.jks",
+      "keystorePassword": "mylesi",
+      "keyAlias": "upload",
+      "keyPassword": "mylesi"
```

Comment:
Those look like a password, but is it safe here ?
I'm afraid of that someone abuses them

Reply:
You're right. I will remove this file from git history and keep it only locally. Thanks for spotting this